src/main/resources/org/jenkinsci/plugins/docker/traceability/core/DockerEventsAction/_api.jelly

```diff
@@ -146,8 +146,60 @@
   Parameters:
   <ul>
     <li>
-      <b>json</b> - JSON data being generated by an external script. 
-      The format is described in <b style="color:red">TODO: reference to Wiki or external docs</b>.
+      <b>json</b> - JSON structure being generated by an external script, which 
+          includes all the available data regarding the container.
+          This structure has the following entries:
+      <dl>
+        <dt>event</dt>
+        <dd>Describes the event, for which the record is being submitted.
+            This entry is mandatory, but its status may have the &quot;NONE&quot; 
+            value, which will notify Docker Traceability Plugin that there's
+            no status changes.
+            <br/><b>Data format:</b> Single JSON entry similar to the ouput of the
+            <a href="https://docs.docker.com/reference/api/docker_remote_api_v1.16/#monitor-dockers-events">
```

Comment:
BTW `target="_blank"` (IIRC) is helpful for links shown inside the Jenkins web interface, though it is most important in forms because Jenkins does not handle the _Back_ button well.